### DISCUSSION TITLE
Visualize OpenDTU in Netdata

### DISCUSSION THREAD

**Discussion by civilengjoiner** [tbnobody/OpenDTU#879] [Q&A]
I want to monitor and visualize the production of solar energy (Power, Yieldday, Yieldtotal) using the opendtu and Netdata which is running on my server.
I have no trouble getting the values either using curl/jason in bash or using mosquitto/mqtt but i fail to understand how to import the collected data into Netdata either using pandas.
I am new to programming.

**Comment by Rosetteonfire92** [comment]
use Grafana with database-access.
